web/ui/react-app/src/hooks/useFetch.ts

```diff
@@ -17,6 +17,12 @@ export const useFetch = <T extends {}>(url: string, options?: RequestInit): Fetc
     const fetchData = async () => {
       setIsLoading(true);
       try {
+        // By adding the current timestamp to the url we are preventing the response from being cached
+        if (includeTime) {
```

Comment:
In which scenario is this needed? The `cache: 'no-store'` option should already disallow any caching, or does that not always work?

Reply:
I was just comparing the old vs new behaviour.

Reply:
Personally, I never experienced any issues with this request being cached. Therefore, I will remove the timestamp from the request